resolvers.js

```diff
@@ -266,20 +267,36 @@ const Resolvers = {
     },
 
     fdrCounts(ds, {inpFdrLvls}) {
-      let outFdrLvls = [], outFdrCounts = [];
+      let outFdrLvls = [], outFdrCounts = [], maxCounts = 0, dbName = '';
       if(ds._source.annotation_counts && ds._source.ds_status === 'FINISHED') {
-        let inpAllLvlCounts = ds._source.annotation_counts[0].counts;
-        inpFdrLvls.forEach(lvl => {
-          let findRes = inpAllLvlCounts.find(lvlCount => {
-            return lvlCount.level === lvl
+        let annotCounts = ds._source.annotation_counts;
+        let dbList = ds._source.ds_config.databases.map(el => el.name);
+        let filteredDbList = annotCounts.filter(el => {
+          return dbList.includes(el.db.name)
+        });
+        for (let db of filteredDbList) {
+          let buzz = db.counts.find(lvlObj => {
+            return lvlObj.level === 10
```

Comment:
I'd suggest to rename `foo` and `buzz` to something more meaningful
Also, this hardcoded `10` may be not the best solution. Consider changing the signature of the endpoint to have this number as a parameter